diff --git a/0x01-variables_if_else_while/0-positive_or_negative.c b/0x01-variables_if_else_while/0-positive_or_negative.c
index 30ae365..b3ff007 100644
--- a/0x01-variables_if_else_while/0-positive_or_negative.c
+++ b/0x01-variables_if_else_while/0-positive_or_negative.c
@@ -1,11 +1,11 @@
 #include <stdlib.h>
 #include <time.h>
 #include <stdio.h>
+
 /**
- * main - main block
- * Description: Get a random number and print the number
- * and if it is positive, negative, or zero
- * Return: 0
+ *main -Determines if a number is positive,negative or zero.
+ * 
+ * Return: Always 0 (Success)
  */
 int main(void)
 {
@@ -14,10 +14,15 @@ int main(void)
 	srand(time(0));
 	n = rand() - RAND_MAX / 2;
 	if (n > 0)
+	{
 		printf("%i is positive\n", n);
+	}
 	else if (n < 0)
-		printf("%i is negative\n", n);
-	else
+	{
 		printf("%i is zero\n", n);
+	}
+	else
+		printf("%i is negative\n", n);
+	}
 	return (0);
 }
